web/src/beta/components/TabMenu/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC, useState, ReactNode } from "react";
+
+import Icon from "@reearth/beta/components/Icon";
+import Icons from "@reearth/beta/components/Icon/icons";
+import { styled } from "@reearth/services/theme";
+
+interface TabObject {
+  icon: keyof typeof Icons;
+  component: ReactNode;
+}
+
+type Props = {
+  title: string;
+  tabs: Record<string, TabObject>;
+};
+
+const TabMenu: FC<Props> = ({ tabs, title }) => {
+  const [selected, setSelected] = useState<string | null>(null);
+
+  return (
+    <Wrapper>
+      <Title>{title}</Title>
+      <Tabs>
+        {Object.entries(tabs).map(([tab, val]) => (
+          <Icon
+            key={tab}
+            onClick={() => setSelected(tab)}
+            icon={val.icon}
+            alt="icon"
+            size={20}
+            style={{
```

Comment:
Please avoid inline styles as much as possible.

Reply:
The icon accepts a style component, that's why I was able to use it. I'll think of some other way if this is not correct.

Reply:
Unless impossible, its nicer to have all custom styles inside a styled component, so just do `const StyledIcon = styled(Icon)`..styles..`

Reply:
Done!